docs/chapters/04_features.md

```diff
@@ -487,6 +487,52 @@ mutation {
 
 > [!NOTE]  Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
 
+
+### Validating read models' filters
```

Comment:
This `before` hooks can be used to more things than just validating, so I would suggest calling this:
```suggestion
### Adding "before" hooks to your read models
```